migrations/settings/0001-migrate-categories-setting.js

```diff
@@ -0,0 +1,31 @@
+const VALID_TARGETS = ["all", "no_sub", "only_sub"];
+
+export default function migrate(settings, helpers) {
+  if (settings.has("categories")) {
+    const categories = settings.get("categories");
+    const newCategories = [];
+
+    categories.split("|").map((item) => {
+      const [categoryName, target] = item.split(":");
+      const categoryId = helpers.getCategoryIdByName(categoryName);
+
+      if (categoryId) {
+        const category = {
+          category_id: [categoryId],
```

Comment:
This is a little awkward because we only support an array of valid category ids for now but is mostly a developer experience concern so I'm willing to defer this for now.